main.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/kirikami/go_exercise_api/config"
+	"github.com/kirikami/go_exercise_api/database"
+	mw "github.com/kirikami/go_exercise_api/middleware"
+	"github.com/kirikami/go_exercise_api/routes"
+	"github.com/labstack/echo"
+	"github.com/labstack/echo/engine/standard"
+	"github.com/labstack/echo/middleware"
+
+	_ "github.com/uber-common/zap"
+)
+
+var (
```

Comment:
@Kirikami this is bad approach, please create `type App struct` and add `Config` and `DB` fields. Then add method `InitConfiguration`, `InitDatabase`